cloudoptim/qptestflowcore
# Dev stepdata testflowdata

Testflow API V2

1. Get By Id: https://documenter.getpostman.com/view/8434357/TzRX7Q9S#7677167b-1f07-418a-9594-11b81629feca
2. Search: https://documenter.getpostman.com/view/8434357/TzRX7Q9S#7677167b-1f07-418a-9594-11b81629feca
3. Create: https://documenter.getpostman.com/view/8434357/TzRX7Q9S#1e69708a-6e79-4478-a9f4-35464d13f255
4. Update: https://documenter.getpostman.com/view/8434357/TzRX7Q9S#f0170a71-7a03-49aa-b26b-e0fa751b25fd
5. Update Status by Step Id: https://documenter.getpostman.com/view/8434357/TzRX7Q9S#80b7af2a-e290-4649-9fab-1d25e0a754b0
6. Check locking: https://documenter.getpostman.com/view/8434357/TzRX7Q9S#4c47c55d-9242-4aaa-b089-608cfd9cefbc
7. Get chart summary by Area: https://documenter.getpostman.com/view/8434357/TzRX7Q9S#9cb00eef-7a9d-4932-81de-ca14bc1705eb
8. Get chart summary by assignment: https://documenter.getpostman.com/view/8434357/TzRX7Q9S#0bcd8d7b-ee48-451b-946a-14097f3fcedc
9. Unlock testflow: https://documenter.getpostman.com/view/8434357/TzRX7Q9S#4af5fb68-f5d4-46e9-809e-cb12c3c12fd7
10. Lock: https://documenter.getpostman.com/view/8434357/TzRX7Q9S#c6f21905-e7cb-4368-974d-44a1cce702f0
